dmutils/content_loader.py

```diff
@@ -413,6 +404,27 @@ def _get_single_question_data(self, form_data):
 
         return {self.id: value if value else None}
 
+    def get_error_messages(self, errors):
+        error_fields = set(errors.keys()) & set(self.form_fields)
+        if not error_fields:
+            return {}
+
+        for field_name in sorted(error_fields):
+            question = self.get_question(field_name)
+            message_key = errors[field_name]
+
+            validation_message = question.get_error_message(message_key, field_name)
+
+            error_key = question.id
+            if message_key == 'assurance_required':
+                error_key = '{}--assurance'.format(error_key)
+
+            return {error_key: {
```

Comment:
It looks like this will return after finding one error message? Are there questions (multiquestions?) that could have multiple error messages that need to be collected before returning? Or will it only ever be one error per question, in which case this should be OK?